internal/xds/translator/testdata/out/xds-ir/ratelimit-disable-headers.listeners.yaml

```diff
@@ -0,0 +1,43 @@
+- address:
+    socketAddress:
+      address: 0.0.0.0
+      portValue: 10080
+  defaultFilterChain:
+    filters:
+    - name: envoy.filters.network.http_connection_manager
+      typedConfig:
+        '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+        commonHttpProtocolOptions:
+          headersWithUnderscoresAction: REJECT_REQUEST
+        http2ProtocolOptions:
+          initialConnectionWindowSize: 1048576
+          initialStreamWindowSize: 65536
+          maxConcurrentStreams: 100
+        httpFilters:
+        - name: envoy.filters.http.ratelimit
+          typedConfig:
+            '@type': type.googleapis.com/envoy.extensions.filters.http.ratelimit.v3.RateLimit
+            domain: first-listener
+            rateLimitService:
```

Comment:
compare this to:

https://github.com/envoyproxy/gateway/blob/329aafc746d0f01df990365b96a09daf4f2fdeac/internal/xds/translator/testdata/out/xds-ir/ratelimit.listeners.yaml#L17-L26

the config `headers.disableRateLimitHeaders: true` causes the line

     enableXRatelimitHeaders: DRAFT_VERSION_03

to be omitted, because OFF is the zero value and marshalling is configured as `omitempty`